services/director-v2/tests/integration/02/utils.py

```diff
@@ -529,13 +529,19 @@ async def _port_forward_legacy_service(  # pylint: disable=redefined-outer-name
     # Legacy services are started --endpoint-mode dnsrr, it needs to
     # be changed to vip otherwise the port forward will not work
     result = run_command(f"docker service update {service_name} --endpoint-mode=vip")
-    assert "verify: Service converged" in result
+    assert (
+        "verify: Service converged" in result
+        or f"verify: Service {service_name} converged" in result
```

Comment:
not sure I get what this modification is about?

Reply:
The tests were failing here. This is just parsing docker logs and it seems the logs changed. I don't know why this is the case... 🤷🏻